### PR TITLE
Add create & read entity checks

### PR DESCRIPTION
**Pull request by greatbacon**
This PR adds `checkCommunityAccess` checks to our two read entity services, & adds a policy permission check for the create event.

### CODE REVIEW

File: src/lib/vocab/entity/entityServices.ts [hunk 1]
```diff
@@ -11,16 +11,22 @@ import {
 } from '$lib/vocab/entity/entityEvents';
 import {toTieEntityIds} from '$lib/vocab/tie/tieHelpers';
 import type {Tie} from '$lib/vocab/tie/tie';
-import {cleanOrphanedEntities} from './entityHelpers.server';
+import {cleanOrphanedEntities} from '$lib/vocab/entity/entityHelpers.server';
+import {checkCommunityAccess, checkPolicy} from '$lib/vocab/policy/policyHelpers.server';
+import {permissions} from '../policy/permissions';
```
Comment (ryanatkn) [inline src/lib/vocab/entity/entityServices.ts:16]:
```suggestion
import {permissions} from '$lib/vocab/policy/permissions';
```

**Review by ryanatkn — COMMENTED** [review]
Looks good, I think I got the tests fixed in #689

**Review by ryanatkn — APPROVED** [review]

File: src/lib/vocab/community/CommunityRepo.ts [hunk 2]
```diff
@@ -113,4 +113,19 @@ export class CommunityRepo extends PostgresRepo {
 		if (!data.length) return NOT_OK;
 		return {ok: true, value: data[0]};
 	}
+
+	async findByEntity(entity_id: number): Promise<Result<{value: Community}>> {
```
Comment (ryanatkn) [inline src/lib/vocab/community/CommunityRepo.ts:117]: What do you think of simplifying this and moving it to the space repo? We're only using the returned `community_id` at the call sites

Comment (greatbacon) [reply src/lib/vocab/community/CommunityRepo.ts:117]: I think that makes sense!

**Pull request merged**